firmware/Sources/HAL/HAL_GPIO/hal_gpio_test.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * TestingGpio.h
+ *
+ *  Created on: 25 may. 2023

```

Comment:
update this header to the standard

Reply:
File deleted as it´s duplicated, the correct one is in firmware/Sources /Test/HAL/